.github/workflows/docs.yml

```diff
@@ -0,0 +1,15 @@
+name: docs 
+on:
+  push:
+    branches:
+      - master
```

Comment:
```suggestion
      - develop
```

Reply:
The HTML documentation is the official one. So it is better that using `master` which is the released version.

Reply:
thanks!